specs/clustertasks/clustertask-multiarch.spec

```diff
@@ -45,7 +45,7 @@ Steps:
       |1   |jib-maven-pz-run |successful|no                     |
 
 ## kn-apply pipelinerun: PIPELINES-17-TC03
-Tags: e2e, linux/amd64, clustertasks, non-admin, kn-apply, sanity
+Tags: e2e, linux/amd64, clustertasks, non-admin, kn-apply
```

Comment:
Shouldn't we have at least one `sanity` test case in `clustertask-multiarch.spec`?

Reply:
Fair point, I would suggest to have `sanity` tag on `jib-maven` tests instead of kn/kn-apply because
1. it will run 1 test instead of 2
2. if we decide to switch our CI to `arm64`, we wouldn't run any test

Reply:
:heavy_plus_sign: